pkg/apis/extensions/v1alpha1/types_containerruntime.go

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ Object = (*ContainerRuntime)(nil)
+
+// ContainerRuntimeResource is a constant for the name of the Container Runtime Extension resource.
+const ContainerRuntimeResource = "ContainerRuntime"
+
+// CRINameWorkerLabel is the name of the label describing the CRI name used in this node.
+const CRINameWorkerLabel = "worker.gardener.cloud/cri-name"
+
+// ContainerRuntimeNameWorkerLabel is a label describing a Container Runtime which should be supported on the node.
+const ContainerRuntimeNameWorkerLabel = "containerruntime.worker.gardener.cloud/%s"
```

Comment:
Put all constants in a const block
```golang
const (
...
)
```

```suggestion
const (
// ContainerRuntimeResource is a constant for the name of the Container Runtime Extension resource.
ContainerRuntimeResource = "ContainerRuntime"
// CRINameWorkerLabel is the name of the label describing the CRI name used in this node.
CRINameWorkerLabel = "worker.gardener.cloud/cri-name"
// ContainerRuntimeNameWorkerLabel is a label describing a Container Runtime which should be supported on the node.
ContainerRuntimeNameWorkerLabel = "containerruntime.worker.gardener.cloud/%s"
)
```